src/components/SearchInput.jsx

```diff
@@ -8,22 +8,23 @@ class SearchInput extends React.Component {
       searchTerm: "",
     }
     this.handleChange = this.handleChange.bind(this)
+    this.handleClick = this.handleClick.bind(this)
   }
 
   handleChange(event) {
     this.setState({searchTerm: event.target.value})
-    console.log(this.state.searchTerm)
   }
 
   handleClick(event) {
     event.preventDefault()
+    console.log(this.state.searchTerm)
```

Comment:
When I use the `Amuse me` button, I'm able to log state to the console. 

I'm really struggling on how to pass it either as props, or as a variable to import into App.js.